### ISSUE TITLE
Unknown type encountered: time-input-row.

### ISSUE THREAD

**Issue by brianhanifin** [home-assistant/home-assistant-polymer#4729]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

`time-input-row` is giving an error in 0.105.0b4.  _"Unknown type encountered: time-input-row."_

## Expected behavior

A field to enter the time should appear.

## Steps to reproduce

1. Upgrade Home Assistant to 0.105.0b4
2. Open a Lovelace page with a time-input-row entity

## Environment

- Home Assistant release with the issue: 0.105.0b4
- Last working Home Assistant release (if known): 0.104.3 (maybe, I cannot confirm this right now)
- UI Type (States or Lovelace): Lovelace
- Browser and browser version: Firefox 72.0.2 / Chrome 79.0.3945.130
- Operating system: Windows 10 64-bit Home v1903 (build 18362.592)

## Problem-relevant configuration

My UI is set in YAML mode.

```yaml
- type: entities
  title: Experimental Sensors
  entities:
    - type: time-input-row
      entity: input_datetime.wakeup_brian_time
      icon: mdi:weather-sunny
```

## Javascript errors shown in your browser console/inspector

```txt
Custom Sidebar is loading custom-sidebar.js:1:9
Custom Sidebar is loaded custom-sidebar.js:17:21
Loading module from “http://hassos.local:8123/community_plugin/config-template-card/config-template-card.js” was blocked because of a disallowed MIME type (“application/octet-stream”).
home
Loading failed for the module with source “http://hassos.local:8123/community_plugin/config-template-card/config-template-card.js”. home:1:1
uncaught exception: http://hassos.local:8123/community_plugin/config-template-card/config-template-card.js
BAR-CARD
Version: 1.6.2 bar-card.js:1228:9
  BUTTON-CARD  
 Version 3.1.1 button-card.js:1431:11
@babel/polyfill is loaded more than once on this page. This is probably not desirable/intended and may have consequences if different versions of the polyfills are applied sequentially. If you do need to load the polyfill more than once, use @babel/polyfill/noConflict instead to bypass the warning. custom-header.js:145:153488
  CUSTOM-HEADER  
  Version 1.3.1  custom-header.js:145:153836
CARD-TOOLS 2 IS INSTALLED
  DeviceID: 2cea05d9-ea8f87f5 card-tools.js:1:13737
 MINI-GRAPH-CARD  0.9.1 mini-graph-card-bundle.js:1:74524
simple-thermostat: 0.33.0 simple-thermostat.js:1:33451
uncaught exception: http://hassos.local:8123/community_plugin/config-template-card/config-template-card.js
```

## Additional information
* I primarily use Firefox, but I cleared the cache in Chrome, then opened the page and the same error persists.
* I am running "Home Assistant" (formerly Hass.io 😉) so I tried opting out of the beta to reinstall 0.104.3. But... the data may be incompatible or something because that entity and many others are reporting "This entity is currently unavailable...".

I will see if I have a snapshot old enough to still have 0.104.3 as it was. I will report back either way.

**Comment by VDRainer** [comment]
Maybe related? https://github.com/home-assistant/home-assistant-polymer/issues/4670
Are your custom-card up-to-date? Especially card-tools?

**Comment by iantrich** [comment]
That's not a valid row type. Just remove the type and it should be interested properly

**Comment by brianhanifin** [comment]
I just restored a 0.104.3 snapshot and those are back

**Comment by brianhanifin** [comment]
> That's not a valid row type. Just remove the type and it should be interested properly

Oh! Huh. It worked until the beta. I took Thomas Loven's statement "time-input-row is now a standard feature of lovelace" as meaning I had to remove the `custom:` from that type, and Lovelace didn't disagree with me until now.

I will switch back to the beta to verify this fixes it. You will not hear back from me unless it does not... but I now see there is no time-input-row listed on the lovelace page. Thanks @iantrich .
